src/clj/fluree/db/json_ld/policy/enforce.cljc

```diff
@@ -52,6 +52,19 @@
     (get-in policy-map [const/iri-modify :default])
     (get-in policy-map [const/iri-view :default])))
 
+(defn policies-for-flake
+  [{:keys [policy namespace-codes] :as db} [s p o :as flake] modify?]
+  (let [s-iri (iri/sid->iri s namespace-codes)
```

Comment:
Why do this for every flake? If the `s-targets` is already encoded, this conversion isn't required at all.

Reply:
Good point, I can easily move SID encoding to the parsing stage.